src/clj/game/core/commands.clj

```diff
@@ -401,7 +401,8 @@
      state side
      {:prompt "Choose a card to derez"
       :choices {:card #(rezzed? %)}
-      :effect (effect (derez target))}
+      :async true
+      :effect (effect (derez eid target {:no-event true}))}
```

Comment:
this and the other derez command should probably also not trigger checkpoints, right?

Reply:
hmm, I think the checkpoint is probably fine (if only because it will force game values to be correct and force any lingering effects to be processed)